### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by Umuhan**
In this pull request i did:
- [ ] Add works section.
- [ ] Add about section. 
- [ ] Add style for both sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,6 +79,177 @@ <h1 class="heading-title headline-title-1">
       </div>
     </section>
 
+    <!-- My Recent works -->
+    <section class="recent-work">
+      <div class="container">
+        <div class="recent-work-header">
+          <h2 class="heading-title">My Recent works</h2>
```
Comment (RaoAkif) [inline index.html:86]: [OPTIONAL] You can capitalize the word "works" from the heading "My Recent Works", to stay consistent with the Figma file. It seems not a big issue, but being consistent in overall font decisions make the design beautiful.

File: css/style.css [hunk 2]
```diff
@@ -91,3 +91,156 @@
...
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 0.8rem;
+  margin-bottom: 2rem;
+}
+
+.lang {
+  font-weight: 600;
+  font-size: 0.9rem;
```
Comment (RaoAkif) [inline css/style.css:157]: As per the Figma file, the size of this `lang` tag text is `1.2rem`, but in you design it is `font-size: 0.9rem;`. Kindly update the font-size and stay consistent with the provided FIGMA design.

File: css/style.css [hunk 3]
```diff
@@ -91,3 +91,156 @@
...
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 0.8rem 1.2rem;
+}
+
+.btn {
+  padding: 1.2rem 2.4rem;
+  border-radius: 4px;
+  font-weight: 500;
+  font-size: 1.8rem;
```
Comment (RaoAkif) [inline css/style.css:170]: As per the Figma file, the size of this `btn` class font text is 1.7rem, but in you design it is font-size: 1.8rem;. Kindly update the font-size and stay consistent with the provided FIGMA design.

File: index.html [hunk 4]
```diff
@@ -79,6 +79,177 @@ <h1 class="heading-title headline-title-1">
...
+            <li class="lang-about">RSpec</li>
+            <li class="lang-about">Capybara</li>
+            <li class="lang-about">Salenium</li>
+          </ul>
+        </div>
+        <div class="about-me-box">
+          <img
+            class="about-img"
+            src="./images/icon-skills.svg"
+            alt="Language icon"
```
Comment (RaoAkif) [inline index.html:239]: The size of these section in Figma design is
```
height: 316px;
width: 327px;
```

But you have put the size as:
```
height: 325px;
width: 224px;
```
Kindly update the design, as per the design given to stay consistent.

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @Umuhan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Umuhan) [reply index.html:239]: Done

File: css/style.css (hunk 3 above)
Comment (Umuhan) [reply css/style.css]: Done.

Comment (Umuhan) [reply css/style.css]: Done.

File: index.html (hunk 1 above)
Comment (Umuhan) [reply index.html]: Done.

File: css/style.css (hunk 2 above)
Comment (Umuhan) [reply css/style.css]: Done.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Umuhan 

Your project is complete! Great job for making the changes suggested by the previous reviewer 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Figma design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**